app/models/manageiq/providers/hawkular/middleware_manager/middleware_domain.rb

```diff
@@ -0,0 +1,4 @@
+module ManageIQ::Providers
```

Comment:
The idea is to have a domain model that is available for other providers of the same kind.
E.g. someone could write a OracleWebLogic Middleware provider that has the same base entities as the Hawkular one.